proposals/Exceptions.md

```diff
@@ -178,58 +190,40 @@ there to query.
 
 ## Changes to the binary model
 
-This section describes changes in the binary encoding design document (ref?).
-
-### Language types
-
-A new type constructor '0x50' for exception types will be added.
-
-All types are distinguished by a negative `varint7` values that is the first
-byte of their encoding (representing a type constructor):
-
-| Opcode | Type constructor |
-|--------|------------------|
-| `-0x01` (i.e., the byte `0x7f`) | `i32` |
-| `-0x02` (i.e., the byte `0x7e`) | `i64` |
-| `-0x03` (i.e., the byte `0x7d`) | `f32` |
-| `-0x04` (i.e., the byte `0x7c`) | `f64` |
-| `-0x10` (i.e., the byte `0x70`) | `anyfunc` |
-| `-0x20` (i.e., the byte `0x60`) | `func` |
-| `-0x30` (i.e., the byte `0x50`) | `except` |
-| `-0x40` (i.e., the byte `0x40`) | pseudo type for representing an empty `block_type` |
+This section describes changes in
+the
+[binary encoding design document](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md).
 
 ### Exception type signatures
 
-An exception type is described by its exception type signature as follows:
-
-`except_type`
+An exception is described by its exception type signature, which corresponds to
+the function signature of the exception constructor. Hence, an exception _type
+signature_ is a `func_type` that returns zero values.
```

Comment:
I would prefer not to reuse function types for exceptions, since they don't really fit. They don't "compute" and they don't have a return type, not even an empty one. It rather is a tuple type.

I think it's fine to define exception types inline as simply a vector of value types, instead of an indirection through the type section.